guide/ops/troubleshooting/deployment.md

```diff
@@ -143,3 +152,27 @@ start tasks are completed).
 
 See the [overview](overview.html) for where to find additional information, especially the section on
 "Entity's Error Status".
+
+## Invalid packet error
+
+If you receive an error message similar to the one below when provisioning a VM, you can workaround the issue by explicitly setting the user that AMP should use to login to the VM (typically the OS default user).
```

Comment:
I'd reword to something like:

    If you receive an error message similar to the one below when provisioning a VM, it means that the wrong 
    username is being used for ssh'ing to the machine. The "invalid packet" is because a response such as 
    "Please login as the ubuntu user rather than root user." is being sent back.
    
    You can workaround the issue by explicitly setting the user that AMP should use to login to the VM 
    (typically the OS default user).